app/src/pages/auth/InitiatePasswordResetForm.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useCallback, useState } from "react";
+import { Controller, useForm } from "react-hook-form";
+import { css } from "@emotion/react";
+
+import { Alert, Button, Form, TextField, View } from "@arizeai/components";
+
+type InitiatePasswordResetFormParams = {
+  email: string;
+};
+
+export function InitiatePasswordResetForm() {
+  const [message, setMessage] = useState<string | null>(null);
+  const [error, setError] = useState<string | null>(null);
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+  const onSubmit = useCallback(
+    async (params: InitiatePasswordResetFormParams) => {
+      setMessage(null);
+      setError(null);
+      setIsLoading(true);
+      try {
+        const response = await fetch("/auth/initiate-password-reset", {
+          method: "POST",
+          headers: {
+            "Content-Type": "application/json",
+          },
+          body: JSON.stringify(params),
+        });
+        if (!response.ok) {
+          setError("Failed attempt");
```

Comment:
Error is not clear - make the message more clear to the user what might be wrong